client/src/components/OptionsetField/OptionField.scss

```diff
@@ -0,0 +1,5 @@
+.option-field--disabled {
+  label, input[type="radio"], input[type="checkbox"] {
+    cursor: not-allowed;
+  }
+}
```

Comment:
Instead of requiring a CSS class, can we please respect the underlying html prop?

```css
input[disabled], input[readonly] {
	cursor: not-allowed;
}
```

Reply:
some "do not respond to clicks" behaviours only come with the disabled attributes, unfortunately.

that css is fine - but labels wouldn't get the cursor style without the class anyway